lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
   margin-bottom: 0;
+}
 
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
+.nx-card {
+  @include container-vertical;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
```

Comment:
I feel like it might be simpler and more future-proof to use `flex` for this.  By using grid in this way, you're basically assuming the first element will always be the header.  If you use flex, the sizings would be specified by the children, not here, so there wouldn't have to be assumptions about which children are present.

Reply:
> I feel like it might be simpler and more future-proof to use `flex` for this. By using grid in this way, you're basically assuming the first element will always be the header. If you use flex, the sizings would be specified by the children, not here, so there wouldn't have to be assumptions about which children are present.

That's a good point https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/1a904c393841bca80421915a60e0b8e1d7f4305f